knowledge-base/kb-multicolumncombobox-howto-remove-selection-box.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Remove the selection boxes inside RadMultiColumnComboBox
+description: Example showing how to remove the selection boxes of RadMultiColumnComboBox
```

Comment:
You can remove the "Example showing" part from the description. Keep only the most important words, and particles that allows you to keep a meaningful description. 

"How to remove the selection boxes of RadMultiColumnComboBox" is good enough.